cpp/include/cuvs/cluster/kmeans.hpp

```diff
@@ -20,6 +20,7 @@
 #include <raft/core/logger.hpp>
 #include <raft/core/resources.hpp>
 #include <raft/random/rng_state.hpp>
+#include <rapids_logger/logger.hpp>
```

Comment:
Should this usage mean `cuvs` should pick up build and runtime dependencies on `rapids-logger`? Or are we ok with transitively getting it from RAFT because this is the only place and it's in a tightly-coupled-to-RAFT part of the codebase?

Reply:
Yeah that was pretty much my reasoning. The include is itself guaranteed to be pulled in by the raft logger header, which we also have above, and we're passing the verbosity to raft's logger, so I felt OK with this.